www/js/recommendations.js

```diff
@@ -47,10 +50,13 @@ angular.module('emission.main.recommendations',['emission.services', 'emission.p
       CommHelper.getSuggestion().then(function(result) {
         $ionicLoading.hide();
         console.log(result);
-        $scope.name = result.message;
-        $scope.mode = result.method;
+        $scope.message = result.message;
+        $scope.question = result.question;
+        $scope.loc = result.suggested_loc;
+        $scope.mode = "Also try " + result.method + " instead";
         $scope.bid = result.businessid;
         $scope.stars = result.rating;
+        $scope.rating = "img/small/small_"+$scope.stars+".png";
```

Comment:
Isn't this part of #561 ? Why is it showing up here?

Reply:
this is implementing the same change on recommendations.js, it was only implemented on details.js before